app/config/PaginationAppConfig.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config
+
+import com.google.inject.{Inject, Singleton}
+import play.api.Configuration
+
+@Singleton
+class PaginationAppConfig @Inject() (configuration: Configuration) {
```

Comment:
Looks good. One more thought that just occurred to me is that we should add this to the in a module as an eager binding, since for a failure to load this config value, we would want the application to fail booting